community/recruitment.md

```diff
@@ -18,3 +18,5 @@ No. By acceptance, we mean that they will hold a number of responsibilities
 - Be an active member of the society
 
 However, we have been collaborating with non-KOSS members since forever. They are welcome to join/lead our events or pitch something to us. If they want to talk about something, we can have them in one of our meetings as well.
+## How many students do we take?
+Altough the number of students varies from year to year. We try to include limited number of students to presrve the culture of KOSS and not create unessesary chaos without actually having productive work.
```

Comment:
Please run the paragraph through Grammarly once.

Reply:
Could we elaborate on this more? Can we add some bullet points as to discuss about the following points - 
1. Could you add an upper bound of how many folks do we want to restrict ourselves? I feel like 16-18 is a hard stop. Would be difficult to manage more people than that.

2. Why having a lean team makes more sense for us, like more cohesive bonding among teammates, more accountability, makes it easier to spin up and work on ideas more quickly.

Also the changes requested by @thealphadollar . Otherwise LGTM 👍